inc/Engine/Optimization/Minify/CSS/Minify.php

```diff
@@ -135,10 +135,9 @@ private function replace_url( $url ) {
 			$url = rocket_add_url_protocol( $url );
 		}
 
-		$unique_id     = md5( $url . $this->minify_key );
-		$filename      = preg_replace( '/\.(css)$/', '-' . $unique_id . '.css', ltrim( rocket_realpath( $parsed_url['path'] ), '/' ) );
+		$filename      = ltrim( rocket_realpath( $parsed_url['path'] ), '/' );
 		$minified_file = rawurldecode( $this->minify_base_path . $filename );
-		$minify_url    = $this->get_minify_url( $filename, $url );
+		$minify_url    = $this->get_minify_url( $filename, $url ) . '?ver=' . rocket_direct_filesystem()->mtime( $minified_file );
```

Comment:
```suggestion
		$minify_url    = add_query_arg( 'ver', rocket_direct_filesystem()->mtime( $minified_file ), $this->get_minify_url( $filename, $url ) );
```

We can use the built-in WP function to add the query arg to the URL, what do you think?

Reply:
awesome idea.
Done